src/loaders/tiff/ome-tiff.ts

```diff
@@ -2,13 +2,10 @@ import type { GeoTIFF } from 'geotiff';
 import { fromString } from '../omexml';
 
 import TiffPixelSource from './pixel-source';
-import {
-  getOmeLegacyIndexer,
-  getOmeSubIFDIndexer,
-  OmeTiffIndexer
-} from './lib/indexers';
+import { getOmeLegacyIndexer, getOmeSubIFDIndexer } from './lib/indexers';
 import { getOmePixelSourceMeta, guessTileSize } from './lib/utils';
 import type Pool from './lib/Pool';
+import type { OmeTiffIndexer } from './lib/indexers';
```

Comment:
Got an error about the type not being found without this.

Reply:
Went back to try it again and get the actual text of the error and couldn't reproduce so reverted this change.

Reply:
Ah, ok I was looking in the wrong place.  I get

```Uncaught SyntaxError: import not found: OmeTiffIndexer```

in the browser console.